src/ui/EditTask.svelte

```diff
@@ -508,6 +521,19 @@ Availability of access keys:
                 accesskey={accesskey('d')}
             />
 
+            <!-- --------------------------------------------------------------------------- -->
+            <!--  Reminder Date  -->
+            <!-- --------------------------------------------------------------------------- -->
+            <label for="reminder" class="accesskey-first">Reminder</label>
```

Comment:
This is a little tricky...
The class `accesskey-first` says 'mark the first letter as the shortcut'..
However, as all the letters in Reminder are already used as shortcuts, the key `c` is chosen...

Which leaves two questions:
- How do we convey to users that `c` is the access key?
- How do we ensure that we do not mark the wrong character...

Reply:
There is now a pattern for doing this on main, for the Done and Created fields IIRC.